### ISSUE TITLE
Add aria attributes to the Notice_M3 application 

### ISSUE THREAD

**Issue by trystant** [ZaBAOW/Notice_M3#21]
We want to make sure that the Notice_M3 application provides valid discription & data for accessibility, so we'll be adding attributes for aria-live regions as well as the extension overall.

**Comment by trystant** [comment]
This looks like a nice [guide](https://developer.chrome.com/extensions/a11y) for adding the appropriate attributes to HTML elements and to have toolbar activity recognized.

**Comment by ZaBAOW** [comment]
When the popup window is opened, the screen reader will read off every element that exists on the popup window.  As an added bonus it will convey whether the buttons on the window are clickable or not.  Also when there are clickable buttons and the user pressed on the tab key, the button selected will be read by the screen reader.  The next step in making this ARIA approved will be changing the way in which the current threshold is displayed.  I believe that there should be a text field in place where the value is now, so that the screen reader can read the value when the field is selected, and for the user the utilize in setting their own threshold.  The one fix that this portion of the window will need however, is that the field should be restricted to the user if the value is equal to zero so the app has a chance to give the user a general value of messages/min for said channel.  Afterwards, the user is free to change and reset the value.
